src/watchmaker/managers/base.py

```diff
@@ -375,10 +377,27 @@ def _worker_execution(self):
     def _worker_validation(self):
         pass
 
-    @abc.abstractmethod
-    def worker_cadence(self):  # noqa: D102
+    def _before_install(self, workers):
```

Comment:
With the current approach, this `_before_install` method isn't doing anything? Seems reasonable to remove it?